src/pod-utils.ts

```diff
@@ -0,0 +1,20 @@
+import type Pod from './Pod';
+
+// Functions are put here so that they can be mocked for testing purposes.
+
+// eslint-disable-next-line import/prefer-default-export
+export async function getPersonas(
+  pod: Pod,
+  address: string,
+): Promise<Array<{ type: string; address: string }>> {
+  const personas = [];
+  if (pod.isAdmin(address)) personas.push({ type: 'admin', address });
+  if (await pod.isAdminPodMember(address)) personas.push({ type: 'adminPodMember', address });
```

Comment:
this this should be `address: adminPod.address`

Reply:
i think lets validate that

Reply:
yeah you're spot on, have updated this now and will push up in a bit!